nextcloudappstore/core/views.py

```diff
@@ -54,6 +58,13 @@ def get_queryset(self):
             queryset = queryset.filter(categories__id=category_id)
         if featured == "true":
             queryset = queryset.filter(featured=True)
+        if maintainer:
+            try:
+                user = User.objects.get_by_natural_key(maintainer)
+                queryset = queryset.filter(Q(owner=user) |
+                                           Q(co_maintainers=user))
+            except ObjectDoesNotExist:
+                return []
```

Comment:
At least the result is as desired in this case, and I've previously seen that a list can sometimes take the place of a queryset. Would you rather have something like `return QuerySet()` or `queryset = queryset.filter(owner__username__exact=maintainer)`?